### ISSUE TITLE
carriage return in expression ?

### ISSUE THREAD

**Issue by aktxyz** [appsup-dart/expressions#28]
it looks like having a carriage return in an expression does not work, has there been any consideration around an option to allow carriage returns ? this is mainly to allow a more complicated expression to be formatted ... so not crazy important ... but hey we software types tend to like formatting :)

**Comment by rbellens** [comment]
I guess, this could be easily added. I'll add it when I find some time. In the mean time, I think you could simply replace all carriage returns with spaces before parsing: `Expression.parse(stringExpression.replaceAll('\n',' '))`

**Comment by aktxyz** [comment]
Good idea on the spaces ... I am not using the Expression package directly but rather via another package ... but I have been making tweaks to that so will just stuff this in as well ... thanks !
